.circleci/config.yml

```diff
@@ -0,0 +1,175 @@
+version: 2.1
+
+checkout:
+  post:
+    - >
+      if [ -n "$CI_PULL_REQUEST" ]; then
+        PR_ID=${CI_PULL_REQUEST##*/}
+        git fetch origin +refs/pull/$PR_ID/merge:
+        git checkout -qf FETCH_HEAD
+      fi
+
+
+executors:
+  py27:
+    docker:
+      - image: circleci/python:2.7
+
+  wagon_generator:
+    machine:
+      image: ubuntu-1604:201903-01
+
+
+commands:
+  check_py3_compat:
+    steps:
+      - run:
+          name: install futurize
+          command: pip install future --user
+      - run:
+          # those modules contain code that futurize would want to change,
+          # so let's just remove them so that it doesn't report on them
+          # (there's no "exclude" functionality in futurize)
+          name: remove compat modules
+          command: |
+            rm cloudify_gcp/_compat.py
```

Comment:
change this path

Reply:
I'll remove this command- there's no _compat needed for this one.